main.nf

```diff
@@ -749,7 +717,7 @@ process assign_clones{
     set file("${geno.baseName}_clone-pass.tab"), file("$geno_fasta"), val("$id") into ch_for_germlines
     file "${geno.baseName}_clone-pass.tab"
```

Comment:
You define a lot of output files but don't pass them into channels, I wonder if it is necessary to define them at all.